packages/contracts-bedrock/test/L1/OPContractsManager.t.sol

```diff
@@ -855,3 +860,67 @@ contract OPContractsManager_AddGameType_Test is Test {
         );
     }
 }
+
+/// @title OPContractsManager_NoSuperchainOrProtocolVersionsUpgrade_Test
+/// @notice Tests that upgrading worldchain fails when using OPCM V2.0.0,
+///         and then passes when using the modified OPCM.
+contract OPContractsManager_NoSuperchainOrProtocolVersionsUpgrade_Test is OPContractsManager_Upgrade_Harness {
+    IOPContractsManager opcmV200;
+    address worldchainProxyAdmin;
+
+    function setUp() public override {
+        super.setUp();
+
+        // Set the fork block number to 21983965
+        vm.createSelectFork("https://eth.llamarpc.com", 21983965);
```

Comment:
I think that there is probably an env var that we use elsewhere that we should use here instead of this URL. Otherwise, we risk getting rate limited in CI.